pkgs/applications/science/electronics/simulide/default.nix

```diff
@@ -0,0 +1,55 @@
+{ appimageTools, symlinkJoin, lib, fetchurl, makeDesktopItem }:
+
+let
+  pname = "simulide";
+  version = "0.4.15";
+  release = "SR10";
+
+  src = fetchurl {
+    url = "https://launchpad.net/${pname}/${version}/${version}-stable/+download/SimulIDE_${version}-${release}.AppImage";
+    sha256 = "25f85f5262b39c8b25da042f11a0fd517c5d2750e81e538ac15992659ebea03c";
+  };
+
+  binary = appimageTools.wrapType2 {
+    name = pname;
+    inherit src;
+  };
+  # we only use this to extract the icon
+  appimage-contents = appimageTools.extractType2 {
+    name = "SimulIDE_${version}-${release}";
+    inherit src;
+  };
+
+  desktopItem = makeDesktopItem {
+    name = "simulide";
+    exec = "${binary}/bin/simulide";
+    icon = "simulide";
+    comment = "A simple real time electronic circuit simulator, supporting PIC, AVR and Arduino.";
+    desktopName = "SimulIDE";
+    genericName = "SimulIDE";
+    categories = [ "Education" "Electronics" ];
+  };
+
+in
+  symlinkJoin {
+    name = "${pname}_${version}-${release}";
+    paths = [ binary ];
+
+    postBuild = ''
+      mkdir -p $out/share/pixmaps/ $out/share/applications
+      cp ${appimage-contents}/simulide.png $out/share/pixmaps
+      cp ${desktopItem}/share/applications/* $out/share/applications/
+    '';
+
+    meta = with lib; {
```

Comment:
Might be good to add the `mainProgram` on meta.

Reply:
Thanks for the suggestion! I applied the changes and now it's working!!